packages/xo-server/src/xo-mixins/backups.js

```diff
@@ -456,11 +456,9 @@ export default class {
 
       // (Asynchronously) Identify snapshot as future base.
       promise
-        .then(() => {
-          return srcXapi._updateObjectMapProperty(srcVm, 'other_config', {
-            [TAG_LAST_BASE_DELTA]: delta.vm.uuid,
-          })
-        })
+        .then(() =>
+          srcVm.update_other_config(TAG_LAST_BASE_DELTA, delta.vm.uuid)
```

Comment:
what if `vm` is undefined ?

Reply:
`date.vm` will never be undefined by contract, it's the API of `exportDeltaVM`.

Anyway it's out of the scope of this PR :slightly_smiling_face: